templates/header/topbar_begin.html

```diff
@@ -48,12 +49,17 @@
                     </li>{#
 
                     The Rust dropdown menu
-                    #}<li class="pure-menu-item pure-menu-has-children pure-menu-allow-hover pure-menu-opt">
-                        <a href="https://www.rust-lang.org/" target="_blank" class="pure-menu-link">
-                            Rust
+                    #}<li class="pure-menu-item pure-menu-has-children">
+                        <a href="#" class="pure-menu-link" aria-label="Rust">
+                            <span class="title">Rust</span>
```

Comment:
Why not put the title on the `<a>` tag and skip the `<span>`? This title doesn't even show up on hover for me.

Reply:
Just checked and it's indeed unnecessary. Removing it.